R/texreg.R

```diff
@@ -32,10 +33,14 @@ screenreg <- function(l, file = NULL, single.row = FALSE,
   decimal.matrix <- aggregate.matrix(models, gof.names, custom.gof.names, 
       digits, returnobject = "decimal.matrix")
   
-  m <- customnames(m, custom.coef.names)  #rename coefficients
+  if (!is.null(custom.coef.map)) {
+    m <- custommap(m, custom.coef.map)
+  } else {
+    m <- omitcoef(m, omit.coef)  #remove coefficient rows matching regex
+    m <- customnames(m, custom.coef.names)  #rename coefficients
```

Comment:
While you are at it, can you check issue #24 (by @christophergandrud) as well? Not sure if that's still a problem or not.